zlib/conn.go

```diff
@@ -432,7 +429,7 @@ func (c *Conn) SMTPBanner(b []byte) (int, error) {
 }
 
 func (c *Conn) EHLO(domain string) error {
-	cmd := []byte("EHLO " + domain + "\r\n")
+    cmd := []byte("EHLO " + domain + "\r\n")
```

Comment:
It looks like there was a spacing change here, can you make sure to run `go fmt` on your code? (Or modify your text editor to do that and run `goimports` automatically on save).